# HYPOTHESIS TEST

Bombay hospitality Ltd. operates a franchise model for producing exotic Norwegian dinners throughout New England. The operating cost for a franchise in a week (W) is given by the equation W = 1,000 + 5X, where X represents the number of units produced in a week. Recent feedback from restaurant owners suggests that this cost model may no longer be accurate, as their observed weekly operating costs are higher.

**Objective:**
To investigate the restaurant owners' claim about the increase in weekly operating costs using hypothesis testing.
Data Provided:
* The theoretical weekly operating cost model: W = 1,000 + 5X
* Sample of 25 restaurants with a mean weekly cost of Rs. 3,050
* Number of units produced in a week (X) follows a normal distribution with a mean (μ) of 600 units and a standard deviation (σ) of 25 units

### 1. State the Hypotheses statement:

* Null Hypothesis (𝐻o): The mean weekly operating cost is equal to the theoretical cost.
 **Ho: μ=μ0**
* Alternative Hypothesis (Ha): The mean weekly operating cost is higher than the theoretical cost.
 **Ha: μ>μ0** 
* μ is the observed mean weekly cost.
* μ0 is the theoretical mean weekly cost based on the cost model.

### 2. Calculate the Test Statistic:

In [15]:
# import libraries
import pandas as pd
import numpy as np
import scipy.stats as stats
import math
import warnings
warnings.filterwarnings('ignore')

In [16]:
# Given data
sample_mean = 3050  # Rs.
theoretical_mean = 1000 + 5 * 600  # Rs.
sigma = 5 * 25  # 
n = 25  # sample size

In [17]:
# Step 2: Calculate the test statistic
test_statistic = (sample_mean - theoretical_mean) / (sigma / np.sqrt(n))
test_statistic

-38.0

### 3. Determine the Critical Value:

In [18]:
# Z-critical value for one-tailed test
alpha = 0.05
critical_value = stats.norm.ppf(1 - alpha)
critical_value

1.6448536269514722

### 4. Make a Decision:

In [19]:
(test_statistic,critical_value)

(-38.0, 1.6448536269514722)

**Since the test statistic −38.0 is far less than the critical value 1.645, it falls in the rejection region of the null hypothesis.**

### 5. Conclusion:

***Given that the test statistic is significantly lower than the critical value, we fail to reject the null hypothesis. This result suggests that there isn't sufficient evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.***

# CHI-SQUARE TEST

 Mizzare Corporation has collected data on customer satisfaction levels for two types of smart home devices: Smart Thermostats and Smart Lights. They want to determine if there's a significant association between the type of device purchased and the customer's satisfaction level.

### Step 1: State the Hypotheses:
**Null Hypothesis (Ho)**: There is no significant association between the type of smart home device purchased and the customer satisfaction level. In other words, the two variables are independent.

**Alternative Hypothesis (Ha)**: There is a significant association between the type of smart home device purchased and the customer satisfaction level. In other words, the two variables are not independent.

### 2. Compute the Chi-Square Statistic:

In [1]:
import numpy as np
import scipy.stats as stats
# Given data
observed = np.array([
    [50, 70],
    [80, 100],
    [60, 90],
    [30, 50],
    [20, 50]
])
# Step 2: Calculate the expected frequencies
row_totals = observed.sum(axis=1)
col_totals = observed.sum(axis=0)
grand_total = observed.sum()
expected = np.outer(row_totals, col_totals) / grand_total
expected

array([[ 48.,  72.],
       [ 72., 108.],
       [ 60.,  90.],
       [ 32.,  48.],
       [ 28.,  42.]])

In [2]:
# Calculate the Chi-Square statistic
chi_square_statistic = ((observed - expected) ** 2 / expected).sum()
chi_square_statistic

5.638227513227513

### 3. Determine the Critical Value:

In [5]:
# Degrees of freedom: (r - 1) * (c - 1)
df = (observed.shape[0] - 1) * (observed.shape[1] - 1)
df

4

In [6]:
# Determine the critical value from the Chi-Square distribution table
alpha = 0.05
critical_value = stats.chi2.ppf(1 - alpha, df)

chi_square_statistic, critical_value, df

(5.638227513227513, 9.487729036781154, 4)

**Since the Chi-Square statistic 5.64 is less than the critical value 9.49, we fail to reject the null hypothesis.**

### Conclusion
**Based on the analysis, there isn't enough evidence to suggest a significant association between the type of smart home device purchased (Smart Thermostats vs. Smart Lights) and the customer satisfaction level. Therefore, the data supports the conclusion that customer satisfaction is independent of the type of smart home device purchased.**